diff --git a/node_modules/secure-random/lib/secure-random.js b/node_modules/secure-random/lib/secure-random.js
index 96a59d0..81b3bd6 100644
--- a/node_modules/secure-random/lib/secure-random.js
+++ b/node_modules/secure-random/lib/secure-random.js
@@ -17,7 +17,12 @@ if (typeof define !== 'undefined' && define.amd) { //require.js / AMD
 function secureRandom(count, options) {
   options = options || {type: 'Array'}
   //we check for process.pid to prevent browserify from tricking us
-  if (typeof process != 'undefined' && typeof process.pid == 'number') {
+  if (
+    typeof process != 'undefined'
+    && typeof process.pid == 'number'
+    && process.versions
+    && process.versions.node
+  ) {
     return nodeRandom(count, options)
   } else {
     var crypto = window.crypto || window.msCrypto
